### ISSUE TITLE
Harbor install problem "Installation Aborted"

### ISSUE THREAD

**Issue by AyyyMateo** [SkyrimTogether/issues-launcher#90]
## Description
I installed the Harbor-1.0.6.exe then ran the harbor setup. I was installing Harbor to my Program Files(x86) then it started downloading but then stopped like 2/5 of the way through. The text says "Installation Aborted" and "Setup was not Successfully." The green bar is still there and pulsing but does not move. I tried downloading harbor in other places/ folders but still did not work. I did install harbor beforehand but had to delete it since McAfee kept quarantining some files. I then deleted everything Habor related from my pc. If anyone can help me it would be much appreciated.

Oh also I have
* The Elder Scrolls V: Skyrim Special Edition
* Windows 10 x64
![image](https://user-images.githubusercontent.com/77513271/104781024-b7a4dd00-574f-11eb-9c01-e3fc5fd3b287.png)

**Comment by discavery3** [comment]
Hello!
After deinstallation you need to clear registry before next installation. Try use Ccleaner to fix issues in registry.

Best regards,
Sobe

**Comment by TB3-Toransu** [comment]
yeah im having this same issue, I installed before but teh program didnt work so I uninstalled, did you find a fix?
